src/Controller/Admin/ActivityController.php

```diff
@@ -27,9 +32,21 @@ class ActivityController extends AbstractController
      */
     private $events;
 
-    public function __construct(EventService $events)
+    /**
+     * @var ActivityRepository
+     */
+    private $activitiesDb;
```

Comment:
I opted to use `activitiesDb`, because activitiesRepo a shorthand had to be used: activitiesRepository would have been too long, imo. Both db and repo are shorthands, but db is the shorter of the two, while still semantically representing how the variable should be used (: as an intermediate to the database). Therefore, I feel like this doesn't degrade the code quality or readability in any way and this is fine here.

Reply:
While I agree that Repo(Repository) could be used as a synonym for Database, I don't really agree it also works the other way around. As a general convention, and to aid the readability, I would strongly suggest using activitiesRepo. Especially since it's only 2 more characters.